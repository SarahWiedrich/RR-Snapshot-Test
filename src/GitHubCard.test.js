import Container from './GitHubCard.js'
import renderer from 'react-test-renderer'

test('renders a snapshot', () => {
  const tree = renderer.create(<Container/>).toJSON()
  expect(tree).toMatchSnapshot()
})