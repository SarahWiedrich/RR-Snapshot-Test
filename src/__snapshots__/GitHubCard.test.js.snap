// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="container"
>
  <div
    className="card border-info"
  >
    <div
      card-body={true}
      className="card-header"
      mb-0={true}
    >
      About Me
    </div>
    <div
      className="card-body"
    >
      <p
        className="card-text"
      >
        GitHub Username: 
      </p>
      <p
        className="card-text"
      >
        My photo
      </p>
      <p
        className="card-text"
      >
        Blurb about me
      </p>
    </div>
    <img
      alt="hotsprings"
      className="card-img-bottom"
      src="hotsprings.jpg"
      style={
        Object {
          "margin": "0 auto",
          "width": "50%",
        }
      }
    />
  </div>
</div>
`;
